mne/source_estimate.py

```diff
@@ -1370,18 +1370,14 @@ def plot(self, subject=None, surface='inflated', hemi='lh',
         hemi : str, 'lh' | 'rh' | 'split' | 'both'
             The hemisphere to display. Using 'both' or 'split' requires
             PySurfer version 0.4 or above.
-        colormap : str
-            The type of colormap to use.
+        colormap : str | array
```

Comment:
Going back and forth, but I think that's a relatively clean way to do the rest of the docs. Feel free to continue the torture though -- might as well get it right :p